rotkehlchen/exchanges/coinbasepro.py

```diff
@@ -388,6 +388,14 @@ def query_online_deposits_withdrawals(
         account_to_currency = self.create_or_return_account_to_currency_map()
         for entry in raw_movements:
             try:
+                # Check if the transaction has been canceled. If so it should be skipped
+                if 'cancel_code' in entry['details'].keys():
+                    log.warning(
+                        f'Skipping coinbase pro asset_movement {entry} due to it being '
+                        f'a cancelled operation with error code {entry["details"]["cancel_code"]}',
```

Comment:
You include the entire json entry in the log in the start. Better put it in the end and keep the log simpler. Skipping coinbasepro deposit/withdrawal due to not having been completed: {entry}